bin/ansible-connection

```diff
@@ -141,16 +142,30 @@ class ConnectionProcess(object):
             self.shutdown()
 
     def connect_timeout(self, signum, frame):
-        display.display('persistent connection idle timeout triggered, timeout value is %s secs'
-                        % self.connection.get_option('persistent_connect_timeout'), log_only=True)
+        msg = 'persistent connection idle timeout triggered, timeout value is %s secs' % self.connection.get_option('persistent_connect_timeout')
+        display.display(msg, log_only=True)
+        raise Exception(msg)
```

Comment:
Also, ansibot is right, you can't raise an exception here and still call shudown

Reply:
Yes, that's right I missed it completely. Though the issue here is after shutdown there is no active socket over which message can be sen to module side. Looking at alternate ways to implement this functionality.